tests/support/case.py

```diff
@@ -210,8 +210,10 @@ def run_cp(self, arg_str, with_retcode=False, catch_stderr=False):
         arg_str = '--config-dir {0} {1}'.format(self.get_config_dir(), arg_str)
         return self.run_script('salt-cp', arg_str, with_retcode=with_retcode, catch_stderr=catch_stderr)
 
-    def run_call(self, arg_str, with_retcode=False, catch_stderr=False):
-        arg_str = '--config-dir {0} {1}'.format(self.get_config_dir(), arg_str)
+    def run_call(self, arg_str, local=False, with_retcode=False, catch_stderr=False):
```

Comment:
The `local` kwarg addition should go at the end of the list so we don't cause an API error. :)